bot_file.py

```diff
@@ -182,7 +183,7 @@ def stop_listening(self, channel: str, pattern: str) -> None:
         self.notify.forget_pattern(person, pattern)
         self.notify.save_to_file(self.notify_file)
 
-    def time_to_show(self, channel: str, person: str) -> None:
+    def ping_person(self, channel: str, person: str) -> None:
         """ notify a person about a message """
         if channel.startswith('D'):
```

Comment:
What's a channel name that starts with D? DMs? ..looks like yeah, although I couldn't find a documentation that explicitly states that. (closest is various encoded IDs mentioned in https://api.slack.com/methods/chat.postMessage)

:art: Maybe add a comment that we don't support IM channels (for now?)